test/e2e/data/infrastructure-docker/v1alpha3/bases/cluster-with-kcp.yaml

```diff
@@ -52,10 +52,11 @@ metadata:
     kcp-adoption.step2: ""
 spec:
   replicas: ${CONTROL_PLANE_MACHINE_COUNT}
-  infrastructureTemplate:
-    kind: DockerMachineTemplate
-    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
-    name: "${CLUSTER_NAME}-control-plane"
+  machineTemplate:
+    infrastructureRef:
+      kind: DockerMachineTemplate
+      apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+      name: "${CLUSTER_NAME}-control-plane"
```

Comment:
I don't think v1alpha3 templates should be changed

Reply:
Reverted, good catch